app/javascript/components/widgets/weather/semi-circle.jsx

```diff
@@ -1,12 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import cityImage from '@images/buildings.png';
+import pvdImage from '@images/buildings.png';
+import bldImage from '@images/boulder_buildings.svg';
 import { yValue, xValue } from '@lib/circle-math';
 import { timeDiffInMinutes, timeAndDateForTimezone } from '@lib/datetime';
 import { colors } from '@lib/theme';
 import styled, { keyframes } from 'styled-components';
 import MoonPhase from '@weather/moon-phase';
 
+const getCityImage = cityName => {
+  if (cityName === 'Boulder') return bldImage;
+  return pvdImage;
+};
+
+const getCityStyle = fillStyle => {
+  return styled.circle`
+  offset-path: path('${p => p.arcPath}');
+  animation: ${p => p.arcAnimation} 1000ms 1 normal forwards ease-in-out;
+  r: ${p => p.ballRadius};
+  fill: ${fillStyle}`;
```

Comment:
The `${p => p....}` above is the properties pattern from styled components. We can just extend that instead of adding another function call.

Reply:
This should be fixed now!